### PR TITLE
chore: info about dedicated postgres instance

### PR DESCRIPTION
**Pull request by blegesse-git**
part of #[1657](https://github.com/influxdata/granite/issues/1657)

_Describe your proposed changes here._

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/clustered/install/prerequisites.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ We **strongly** recommend that you enable object versioning in your object store
 - **PostgreSQL-compatible database** _(AWS Aurora, hosted Postgres, etc.)_:
   Used to store the InfluxDB catalog
   - Supported PostgreSQL versions: **13.8–14.6**
+  - Ensure that the PostgreSQL instance is dedicated exclusively to InfluxDB purposes to avoid conflicts. This prevents issues due to shared usage with other applications.
```
Comment (sanderson) [inline content/influxdb/clustered/install/prerequisites.md:26]:
```suggestion
  - Ensure that the PostgreSQL-compatible instance is dedicated exclusively to InfluxDB. This avoids conflicts and prevents issues caused by shared usage with other applications.
```

File: content/influxdb/clustered/install/prerequisites.md [hunk 2]
```diff
@@ -51,6 +53,8 @@ in a separate namespace from InfluxDB or external to Kubernetes entirely.
 Running the Catalog database and Object Store in a separate namespace or outside
 of Kubernetes makes management of the InfluxDB instance easier and helps to
 prevents accidental data loss.
+
+While deploying everything in the same namespace is possible for testing or initial setups, it is not recommended for production environments.
```
Comment (sanderson) [inline content/influxdb/clustered/install/prerequisites.md:57]:
```suggestion
While deploying everything in the same namespace is possible for testing or initial setup, it is not recommended for production environments.
```

**Review by sanderson — APPROVED** [review]
Just some minor suggestions and I think it's good to go. Thanks @blegesse-git!

File: content/influxdb/clustered/install/prerequisites.md (hunk 1 above)
Comment (blegesse-git) [reply content/influxdb/clustered/install/prerequisites.md]: done!

**Pull request merged**